cypress/page/address-step.page.ts

```diff
@@ -0,0 +1,22 @@
+class AddressStepPage {
+    private BottonLogin: string;
+    private CartNavigation: string
+
+    constructor() {
+        this.BottonLogin = '#SubmitLogin > span';
+        this.CartNavigation = '.cart_navigation span'
+
+
+    }
+
+    public PressLogin(): void {
+        cy.get(this.BottonLogin).click()
+    }
+
+    public PressNavigation(): void {
```

Comment:
This is referring to click on the proceed to checkout button, let's name the method in a way that indicate that we are proceeding to checkout